pandas/conftest.py

```diff
@@ -1999,6 +1999,14 @@ def using_copy_on_write() -> bool:
     )
 
 
+@pytest.fixture
+def using_infer_string() -> bool:
+    """
+    Fixture to check if Copy-on-Write is enabled.
```

Comment:
docstring looks copy/pasted, needs updating

Reply:
Thx for catching